src/index.ts

```diff
@@ -107,6 +110,70 @@ class SchemaDefinition implements SchemaDefinitionType {
         }
         return obj;
     }
+
+    /**
+     * Split class models into an array or with dependencies
+     * will be in an array after the array it depends on
```

Comment:
I don't understand what this docstring is trying to say

Reply:
I have re-worded it